### PR TITLE
Blogpost update

### PR DESCRIPTION
**Pull request by laisbsc**
adding blogpost

### CODE REVIEW

File: _posts/2020-08-06-comm-open-track.markdown [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+Great question, Padawan!
+
+The aim is to have an independent track for your local community within our comfy conference. So that we encourage engagement from local/small communities everywhere.
+
+## How will it work?
+
+We will allocate rooms with time blocks (min. 2 hours) which will be given to community members who submit a Community Space (CS) proposal.
+The plans for the event are all based on your wishes. It will be your event after all, we will just help you with the setup, space and guidance - if it is required.
+
+## Who is this track for?
```
Comment (Cheukting) [inline _posts/2020-08-06-comm-open-track.markdown:18]: I think it is clearer if you say "##Who is this space for? Shall I submit a proposal?" instead

File: _posts/2020-08-06-comm-open-track.markdown [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+We want to give the small communities the chance to step forward and be part of the Global Python one. After all, we are all under the same open source.
+
+![Mounty Python scene](https://media.giphy.com/media/UZQQ0yZtq5Ihq/giphy.gif)
+
+
+## They say the Devil is in the details - Proposal submission
+### Guidelines
+
+The proposal submission for this special track is super simple. We will need:
```
Comment (Cheukting) [inline _posts/2020-08-06-comm-open-track.markdown:33]: special track -> community space

File: _posts/2020-08-06-comm-open-track.markdown [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  - a basic outline describing the type of event you plan to host - please, be as detailed as possible;
+  - an estimate of the expected capacity (number of attendees) for your ~~Python-party~~ serious event;
+  - the community you are affiliated with (e.g., PyLadies, DjangoGirls, PyData, TransCode, etc).
+  - any other info we should be aware about - be the judge!
+
+As always we want to make sure everything is in agreement with our [Code of Conduct](https://pyjamas.live/coc/), so make sure you have a read and behave! We love everyone and really ruins our day to be 'nasty' to attendees. We will if you misbehave, though.
+
+The approved proposals will receive news from the organising committee with the followup details.
+
+![join us gif](https://media.giphy.com/media/WsMBa68UUh86q2SR0d/giphy.gif)
```
Comment (Cheukting) [inline _posts/2020-08-06-comm-open-track.markdown:45]: I think the actually address of the gif is https://i.giphy.com/media/WsMBa68UUh86q2SR0d/giphy.webp

File: _posts/2020-08-06-comm-open-track.markdown (hunk 1 above)
Comment (laisbsc) [reply _posts/2020-08-06-comm-open-track.markdown:18]: Cool!

File: _posts/2020-08-06-comm-open-track.markdown (hunk 2 above)
Comment (laisbsc) [reply _posts/2020-08-06-comm-open-track.markdown:33]:
```suggestion 
The proposal submission for this community space is super simple. We will need:
```

**Pull request merged**